models/fivetran_platform__connector_daily_events.sql

```diff
@@ -124,20 +127,22 @@ join_event_history as (
 
     from
     spine left join connector_event_history
-        on cast(spine.date_day as date) = connector_event_history.date_day
+        on spine.date_day = connector_event_history.date_day
 
-    group by 1,2,3,4,5,6
+    group by spine.date_day, connector_name, connector_id, connector_type, destination_name, destination_id
 ),
 
 final as (
 
     select *
     from join_event_history
 
-    where cast(date_day as timestamp) <= {{ dbt.current_timestamp_backcompat() }}
+    where date_day <= cast({{ dbt.current_timestamp_backcompat() if target.type != 'sqlserver' else dbt.current_timestamp() }} as date)
 
+    {% if target.type != 'sqlserver' %} -- sql server cant order CTEs
     order by date_day desc
+    {% endif %}
```

Comment:
yeah that's a good question -- i suppose i just wonder if current users of the package have come to expect the ordered models

Reply:
I would actually vote to remove these order by statements. We have chosen to not use order by statements in other packages because it forces additional compute on the query that the customer may ultimately already be applying in their own query on top of the model. Removing these order by statements will help improve the performance of the models altogether. 

We will obviously need to call this out in the CHANGELOG and making this breaking will help protect users who are currently relying on these being sorted at the moment.

Reply:
ah yeah that's a great point re: compute. i'll update!

Reply:
updated